src/docs/markdown/conventions.md

```diff
@@ -32,6 +32,7 @@ The network can be any of the following; ones suffixed with `4` or `6` are IPv4
 - UDP: `udp`, `udp4`, `udp6`
 - IP: `ip`, `ip4`, `ip6`
 - Unix: `unix`, `unixgram`, `unixpacket`
+- File descriptors: `fd`, `fdgram`
```

Comment:
I think a short paragraph to explain what a "file descriptor" means in this context would help